apps/labs/posts/packaging_painpoints_in_2021.md

```diff
@@ -0,0 +1,228 @@
+
+- There's no way to distinguish packages that provide the same binaries, e.g., `turbojpeg` vs. `libjpeg`, both package `libjpeg`
+- Need to separate metadata (especially dependencies) from packages.
+
+_RG: this to me seems like a much smaller issue than with PyPI. The comment was about the ability to patch metadata of published conda packages more easily._
+
+- R community seems slow to adopt it
+
+_RG: I've heard from R users before that conda felt a bit "foreign". In the discussion it was suggested that part of that is that conda is Python-based which can still bubble up to the surface on occasion. And that Mamba, which is written in C++, may find a more positive reception in the R community._
+
+- End-to-end security?
+
+_RG: [Purism](https://puri.sm/) and [this blog post from its devs about security in the software supply chain](https://puri.sm/posts/the-future-of-software-supply-chain-security/) were mentioned._
+
+## Topic 3: Other
+
+- The Conda and PyPA worlds still aren't talking to each other much.
+
+_RG: It's great to see that both the PyPA team and the Conda community are getting better organized. There's now more funding for work on PyPI, Warehouse and Pip; the teams working on it have grown, there's good project management, roadmapping and even Pip user experience research going on. And the Conda community has set up a new organization to align on standards, incubate promising new projects. The next step will hopefully be more productive interactions between those two somewhat separate groups._
+
+- `pip` and `conda` interaction still not great
+    - I'd like a way to have "safe" (e.g., pure Python) packages so you can confidently mix them. `+100`
+    - The pip dependency resolver is not great yet and it also ends up with conflicts with conda.
+
+_RG: True. For doing serious scientific computing, data science and ML type work, you often need the combination, using `conda` for the core libraries, and filling in the gaps with `pip` for packages that are not packaged for conda or installing things from source._
+
+
+## Topic 4: There's always good news too
+
+- [The death of `easy_install`](https://github.com/pypa/setuptools/pull/2544)!!!
+- Mambaforge
+- macos-arm64 support in conda-forge
+
+_RG: It'll be a while before we get wheels for core PyData packages, but conda-forge already has support, see [this blog post](https://conda-forge.org/blog/posts/January 24, 2021
```

Comment:
@rgommers same thing here: https://conda-forge.org/blog/
```suggestion
_RG: It'll be a while before we get wheels for core PyData packages, but conda-forge already has support, see [this blog post](https://conda-forge.org/blog/posts/January 24, 2021)
```

Reply:
Hmm, very odd. It's https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/